frontend/src/components/CommandBar/CommandBar.tsx

```diff
@@ -231,15 +225,12 @@ const SearchBar = ({ form }: { form: FormState<CommandBarSearch> }) => {
 								if (e.code === 'Enter') {
 									e.preventDefault()
 									if (e.metaKey || e.ctrlKey) {
-										searchAttribute(
-											currentAttribute ?? ATTRIBUTES[0],
-											{
-												newTab: true,
-												withDate:
-													selectedDates[0].getTime() !==
-													last30Days.startDate.getTime(),
-											},
-										)
+										searchAttribute(currentAttribute, {
+											newTab: true,
```

Comment:
noticed that the new-tab search does not clear the main tab's command bar search term.
eg. if you cmd+k, type sasha, shift+enter, come back to original tab, cmd+k, still shows sasha

Reply:
This is how Notion does it too. Jay mentioned this as confusing as well though, so I'll reset the form on open.